spec/features/view_peeps_spec.rb

```diff
@@ -10,10 +11,21 @@
   end
 
   scenario 'allows user to view all peeps in a browser' do
-    setup_test_database
-    add_row_to_test_database
+    Peep.create(message: 'This is a peep!')
     visit ('/peeps')
     expect(page).to have_content 'This is a peep!'
   end
-
 end
+
+  # As a Maker
+  # So that I can easily see the latest peeps
+  # I want to see a list of peeps in reverse chronological order
+
+  feature 'lists latest peeps first' do
+    scenario 'list of peeps in reverse chronological order' do
+      add_peep('This is my first peep')
+      add_peep('This is my latest peep')
+
+      expect(page.body.index('This is my latest peep')).to be < page.body.index('This is my first peep')
+    end
```

Comment:
I couldn't work out how exactly to test if something comes before or after something else. This is really succinct way of doing it. Nice.